compiler/luci/import/src/GraphBuilderRegistry.cpp

```diff
@@ -129,6 +129,7 @@ GraphBuilderRegistry::GraphBuilderRegistry()
   CIRCLE_NODE(TOPK_V2, CircleTopKV2GraphBuilder);                                          // 48
   CIRCLE_NODE(TRANSPOSE, CircleTransposeGraphBuilder);                                     // 39
   CIRCLE_NODE(TRANSPOSE_CONV, CircleTransposeConvGraphBuilder);                            // 67
+  CIRCLE_NODE(UNIQUE, CircleUniqueGraphBuilder);                                           // 103
```

Comment:
Please remove below `// BuiltinOperator_UNIQUE = 103,` line

Reply:
Removed the line, thanks.